Modal/modal.css

```diff
@@ -0,0 +1,59 @@
+:root {
```

Comment:
make seperate file for colours that are used everywhere, it will be easy to make changes.